src/partition-sample/src/test/scala/VerifyPartitionSample.scala

```diff
@@ -64,4 +62,9 @@ class PartitionSampleSmokeTests extends TestBase {
       .toDF("Col1", "Col2")
   }
 
+  override def testObjects(): Seq[TestObject[PartitionSample]] = Seq(new TestObject[PartitionSample](
+    new PartitionSample().setMode("Head").setCount(3),
+    makeDF))
```

Comment:
minor style: you can put the makeDF on line above

Reply:
Done